pgml-docs/docs/blog/benchmarks/python_microservices_vs_postgresml/pgbench.sql

```diff
@@ -0,0 +1,3 @@
+SELECT pgml.predict('r2',
+	ARRAY[year, quarter, month, distance, dayofweek, dayofmonth, flight_number_operating_airline, originairportid, destairportid, tail_number]
+) FROM flights_delay_mat WHERE originairportid = 14869 and year = 2019 and month = 8 and dayofmonth = 31 LIMIT 1;
```

Comment:
Similarly, it'd be nice to have batch sizes of 1/10/100/1000 here as well, to see scaling.

Reply:
We don't have a batch API in PG at the moment, but if you want to play unfair, set the limit higher. I think we have about 250 rows for that airport and date.